1987/src/main/resources/application.properties

```
@@ -0,0 +1,2 @@
+spring.application.name=issue-1987
+spring.sleuth.propagation.type=custom
```

Could you please add a new line at the end?
```suggestion
spring.sleuth.propagation.type=custom

```